CONTRIBUTORS.txt

```diff
@@ -1,24 +1,30 @@
 # This file is autogenerated by 'contributors-txt',
+# using the configuration in 'script/.contributors_aliases.json'
 # please do not modify manually
 
-Contributors
-------------
+Mainteners
```

Comment:
```suggestion
Maintainers
```

Reply:
Urgh, costly typo. 2 commits a full release and a regeneration 😄